Blocks/HTTP/impl/posix_server.h

```diff
@@ -258,6 +303,29 @@ class HTTPServerPOSIX final {
     }
   }
 
+  HTTPRoutesScopeEntry DoRegisterHandler(const std::string& path,
+                                         std::function<void(Request)> handler,
+                                         const URLPathParams::Count path_param_count,
+                                         const ReRegisterRoute policy) {
+    if (path.empty() || path[0] != '/' || (path.length() > 1 && path[path.length() - 1] == '/') ||
+        !URL::IsValidPath(path)) {
+      CURRENT_THROW(InvalidHandlerPathException("Invalid handler path '" + path + "'"));
```

Comment:
Got it, you only use `IsValidPath` for input URLs. Good idea :+1: , two quick questions:

1) Why not have a dedicated error message for the 2nd part of the `if`, with the trailing slash?
2) Tests for trailing slashes?